code/datums/station_traits/negative_traits.dm

```diff
@@ -410,3 +410,254 @@
+	trait_to_give = STATION_TRAIT_RADIOACTIVE_NEBULA
+
+	blacklist = list(/datum/station_trait/random_event_weight_modifier/rad_storms)
+	threat_reduction = 30
+
+	intensity_increment_time = 5 MINUTES
+	maximum_nebula_intensity = 1 HOURS + 40 MINUTES
+
+	nebula_color = list(0,0,0,0, 0,2,0,0, 0,0,0,0, 0,0,0,1, 0,0,0,0) //very vibrant green
+	space_light_color = COLOR_VIBRANT_LIME
+	carp_color_override = list(
+		COLOR_CARP_GREEN = 1,
+		COLOR_CARP_TEAL = 1,
+		COLOR_CARP_PALE_GREEN = 1,
+		COLOR_CARP_DARK_GREEN = 1,
+	)
+	///When are we going to send them a care package?
+	VAR_PRIVATE/send_care_package_at = INFINITY
+	///How long does the storm have to last for us to send a care package?
+	VAR_PROTECTED/send_care_package_time = 5 MINUTES
+
+/datum/station_trait/nebula/hostile/radiation/New()
+	. = ..()
+
+	for(var/area/target as anything in get_areas(/area/space))
+		RegisterSignals(target, list(COMSIG_AREA_ENTERED, COMSIG_AREA_INITIALIZED_IN), PROC_REF(on_entered))
+		RegisterSignal(target, COMSIG_AREA_EXITED, PROC_REF(on_exited))
+
+/datum/station_trait/nebula/hostile/radiation/on_round_start()
+	. = ..()
+
+	//Let people order more nebula shielding
+	var/datum/supply_pack/pack = SSshuttle.supply_packs[/datum/supply_pack/engineering/rad_nebula_shielding_kit]
+	pack.special_enabled = TRUE
+
+	//Give robotics some radiation protection modules for modsuits
+	var/datum/supply_pack/supply_pack_modsuits = new /datum/supply_pack/engineering/rad_protection_modules()
+	send_supply_pod_to_area(supply_pack_modsuits.generate(null), /area/station/science/robotics, /obj/structure/closet/supplypod/centcompod)
+
+	//Send a nebula shielding unit to engineering
+	var/datum/supply_pack/supply_pack_shielding = new /datum/supply_pack/engineering/rad_nebula_shielding_kit()
+	if(!send_supply_pod_to_area(supply_pack_shielding.generate(null), /area/station/engineering/main, /obj/structure/closet/supplypod/centcompod))
+		//if engineering isnt valid, just send it to the bridge
+		send_supply_pod_to_area(supply_pack_shielding.generate(null), /area/station/command/bridge, /obj/structure/closet/supplypod/centcompod)
+
+	//Disables radstorms, they don't really make sense since we already have the nebula causing storms
+	var/datum/round_event_control/modified_event = locate(/datum/round_event_control/radiation_storm) in SSevents.control
+	modified_event.weight = 0
+
+///They entered space? START BOMBING WITH RADS HAHAHAHA
+/datum/station_trait/nebula/hostile/radiation/proc/on_entered(area/space, atom/movable/enterer)
+	SIGNAL_HANDLER
+
+	if(!ismovable(enterer))
+		return
+
+	enterer.AddElement(/datum/element/radioactive, range = 0, minimum_exposure_time = NEBULA_RADIATION_MINIMUM_EXPOSURE_TIME)
+	//Don't actually make EVERY. SINGLE. THING. radioactive, just make them glow so people arent killed instantly
+	if(!SSradiation.can_irradiate_basic(enterer))
+		enterer.add_filter("glow_nebula", 2, list("type" = "drop_shadow", "color" = "#66ff33", "size" = 2))
```

Comment:
Define `glow_nebula` since you use it in two places.

Also don't we have defines for `"drop_shadow"`?

Reply:
Turns out we don't